frontend/src/features/properties/filter/IPropertyFilter.ts

```diff
@@ -50,4 +50,6 @@ export interface IPropertyFilter {
   maxMarketValue?: '';
   /** Whether to return properties owned by other agencies. */
   includeAllProperties?: boolean;
+  /** when the surplus properties filter is open */
```

Comment:
Needed to add another field value to the filter so that I could check within the `YupSchema` whether it was being called from the Surplus Filter because otherwise we don't care if they are null